app/src/main/java/com/m/helper/DTO/Keycloak.java

```diff
@@ -0,0 +1,18 @@
+package com.m.helper.DTO;
```

Comment:
where do you get this convention from? It's just I never heard about it before. If it's rarely used then it may cause confusion in the future as people may forget what they mean.

Reply:
I am not sure where I picked it up from, but I feel it was from "domain driven design". With that said I'll rename it to be `configModel`. It would be much clear to someone else reading the example.